db/migrate/20240312092625_add_active_to_application_types.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AddActiveToApplicationTypes < ActiveRecord::Migration[7.1]
+  class ApplicationType < ActiveRecord::Base; end
+
+  def change
+    add_column :application_types, :active, :boolean
```

Comment:
Not for now but wondering if there is any chance we might want to archive or expire an application type and in that case might require an enum for status.

Reply:
@benbaumann95 This is what I've mentioned in stand up, this should be an enum as based on Ed's comments, there are three statuses:  `Active, Inactive, Retired`

Reply:
Updated PR to use a `status` column with `inactive`, `active` and `retired`